docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -214,14 +221,44 @@ LicenseInfo.setLicenseKey(
 );
 ```
 
-The grid checks the key without making any network requests.
+### Security
+
+The MUI X checks the license key without making any network requests. It's air gappped.
+
+The license key is designed to be public, the only thing we ask licensed users is not to proactively shoot out their license key to the world. Exposing the license key publicly in a JavaScript bundle is the expected outcome.
+
+### Validation errors
+
+If the validation of the license key fail, the component displays a watermark in development and production.
+End-users can still use the component.
```

Comment:
```suggestion
End users can still use the component.
```

Reply:
Ok, I could find this https://en.wikipedia.org/wiki/End_user#cite_note-1

<img width="1197" alt="Screenshot 2022-01-12 at 22 26 44" src="https://user-images.githubusercontent.com/3165635/149223931-176304a5-0a9d-460f-819f-ebac5974f05d.png">